src/main/java/frc/robot/posev2/VisionData.java

```diff
@@ -0,0 +1,42 @@
+package frc.robot.posev2;
+
+/**
+ *
+ * @author dcowden
+ */
+public class VisionData {
+    private double lateralOffset  = 0.0;
+    private double verticalOffset = 0.0;
+    private double targetWidth = 0.0;
+
+    public VisionData(double lateralOffset, double verticalOffset){
+        this.lateralOffset = lateralOffset;
+        this.verticalOffset = verticalOffset;
+    }
+
+
+    public double getLateralOffset() {
+        return lateralOffset;
+    }
+
+    public void setLateralOffset(double lateralOffset) {
```

Comment:
i'd remove the setters here-- its the immutable data structure pattern, based on the constructor